pandas/tslib.pyx

```diff
@@ -1388,6 +1388,26 @@ cpdef convert_str_to_tsobject(object ts, object tz, object unit,
 
     return convert_to_tsobject(ts, tz, unit)
 
+def _parse_iso8601(object ts):
+    '''
+    Parse string into Timestamp using iso8601 parser. Used ONLY for
```

Comment:
can you move it to ``testing.pyx``? and maybe call it ``_test_parse_iso8601`` or something to make it completely obvious that it should not be used except in testing.